source/whats-new.txt

```diff
@@ -36,43 +36,118 @@ Upcoming Breaking Changes
   and map-reduce methods on `MongoCollection <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#mapReduce(com.mongodb.client.ClientSession,java.lang.String,java.lang.String)>`__
   are deprecated in MongoDB v4.2 and later. They will be replaced by the
   aggregation framework and removed in a future release.
 - Beginning with v5.0, the {+driver-short+} will require Java 11 or later.
+- Beginning with v5.0, several Stream API methods and classes will be
+  removed. To learn which methods will be removed, see :ref:`version-4.11`.
 
 .. _version-4.11:
 
 What's New in 4.11
 ------------------
 
+.. important:: Deprecation Notice
+
+   - The ``getStats()`` and ``isCapped()`` instance methods of the
+     ``DBCollection`` class are deprecated. The corresponding server
+     commands are deprecated in MongoDB v6.2 and later. Use the ``$collStats`` aggregation
+     pipeline stage to retrieve the information provided by these methods
+     instead. You can run the aggregation as shown in the following code example:
+
+     .. code-block:: java
+
+        collection.aggregate(Arrays.asList(
+                         new Document("$collStats",
+                             new Document("storageStats", new Document()))));
+
+     To determine whether the collection is a capped collection, access the value
+     of the ``storageStats.capped`` field returned by this example aggregation.
+
+     To learn more about the ``$collStats`` aggregation operator, see the
+     :manual:`$collStats (aggregation) </reference/operator/aggregation/collStats/>`
+     Server manual entry.
+
+.. fulfills JAVA-4845: (from Upcoming Breaking Changes) getstats and iscapped deprecated.
+
+   - The following network address-related methods are deprecated:
+
+      - ``getSocketAddress()`` and ``getSocketAddresses()`` methods from the `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__ class
+      - ``getUnixSocketAddress()`` from the `UnixServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/UnixServerAddress.html>`__ class
+
+.. fulfills DOCSP-32942
+
+      - The following Stream API methods and classes are deprecated and are
+        scheduled for removal in v5.0:
+
+        - ``streamFactoryFactory()`` method from ``MongoClientSettings.Builder``
+        - ``getStreamFactoryFactory()`` method from ``MongoClientSettings``
+        - ``NettyStreamFactoryFactory`` class
+        - ``NettyStreamFactory`` class
+        - ``AsynchronousSocketChannelStreamFactory`` class
+        - ``AsynchronousSocketChannelStreamFactoryFactory`` class
+        - ``BufferProvider`` class
+        - ``SocketStreamFactory`` class
+        - ``Stream`` class
+        - ``StreamFactory`` class
+        - ``StreamFactoryFactory`` class
+        - ``TlsChannelStreamFactoryFactory`` class
+
+.. fulfills DOCSP-33033
+
 .. note:: Upgraded Netty Version
```

Comment:
This shouldn't be part of the 4.11 What's New.

Reply:
Should I relocate it or remove it? In [JAVA-5038](https://jira.mongodb.org/browse/JAVA-5038), the fix version was tagged with 4.11.0.

Reply:
Discussed on Slack

Reply:
It can be removed.